src/components/ButtonLink.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
could we just use conditional to check if a link is passed instead of having two separate components?

```
if(props.link)
{
    return (
        <a href={props.link}><button className='btn-class'>{props.text}</button></a>
    )
} else {
    return (
        <div>
            <button className='btn-class'>{props.text}</button>
        </div>
    )
}
```